app/navigation/Navigator.scala

```diff
@@ -150,6 +153,14 @@ class Navigator @Inject()() {
     }
   }
 
+  private def removeWarehouseConfirmPageNavigation(userAnswers: UserAnswers):Call = {
```

Comment:
have all of these flows been tested

Reply:
both yes and not have been tested for this in the navigation spec -> 
![image](https://user-images.githubusercontent.com/75027964/233580661-3d26ca76-8766-45b9-afc9-23f2a1df898f.png)